srcpkgs/dina-font/update

```diff
@@ -0,0 +1 @@
+pattern="Download v\K[\d.]+(?=)"
```

Comment:
dina-font, one of my favourite [outdate-by-repology](https://repology.org/metapackage/fonts:dina/versions). Homepage redirects to diferent domain. Yet on forum near original homepage there is well hidden [newer version](https://www.donationcoder.com/forum/index.php?topic=36049.0) to be available from homepage  soon since 2013.